lib/jna/clink_jna.cc

```diff
@@ -151,9 +153,9 @@ OneHotEncoderModel_loadFromMemory(const char *params_str,
   nlohmann::json params = nlohmann::json::parse(params_str);
   std::string is_droplast = params["dropLast"].get<std::string>();
   model->setDropLast(is_droplast != "false");
-  CLINK_JNA_HANDLE_ERROR(
-      model->setModelData(std::string(model_data_str, model_data_str_len)))
-
+  auto error =
```

Comment:
Would it be more intuitive to name it `output` or `maybe_error`?

Reply:
I think `maybe_error` is better. I'll change it this way.